### PR TITLE
Adding duplex session tests

### PR DESCRIPTION
**Pull request by KKhurin**
Had to disable and modify a few tests because of #1962 #1959.

### CODE REVIEW

**Comment by zhenlan** [comment]
@dotnet-bot please run OuterLoop Selfhost Windows_NT Debug

**Comment by KKhurin** [comment]
@dotnet-bot please run OuterLoop Selfhost Windows_NT Debug

File: src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/SessionTests.cs [hunk 1]
```diff
@@ -203,4 +204,347 @@ static ChannelFactory<T> CreateChannelFactoryHelper<T>(string url)
         binding.Security.Mode = SecurityMode.None;
         return new ChannelFactory<T>(binding, helloEndpoint);
     }
+
+    // Without CallbackBehaviorAttribute (and ConcurrencyMode = ConcurrencyMode.Multiple) support
+    // a few duplex session tests are not going to be able to call the service from within our callback. 
+    // This is tracked by issue #1959
+
+    //[CallbackBehavior(ConcurrencyMode = ConcurrencyMode.Multiple)]
+    public class SessionTestsDuplexCallback : ISessionTestsDuplexCallback
```
Comment (StephenBonikowsky) [inline src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/SessionTests.cs]: Files ending in "Tests" as a general rule should only contain tests. Please put all helper and/or common code in a separate file such as SessionTestHelpers.cs or SessionTestCommon.cs

**Comment by StephenBonikowsky** [comment]
Did you run all the Scenario tests before creating the PR? Did they pass?

**Comment by KKhurin** [comment]
@StephenBonikowsky  I would have not submitted a PR if the scenario tests didn't pass.

**Pull request merged**